src/components/message/MessageView.tsx

```diff
@@ -1,261 +0,0 @@
-import { useEffect, useState } from "react";
```

Comment:
so we don't need it anymore?..

Reply:
We don't use the MessageView at all, everything is rendered from the MessageList.